libflux/src/flux/semantic/bootstrap.rs

```diff
@@ -376,12 +376,13 @@ mod tests {
                 String::from("x") => parse("forall [] int")?,
             }),
         };
+        let prelude: HashMap<String, PolyType> = HashMap::new();
```

Comment:
You didn't change the signature of `infer_pkg` so you don't need to create a temporary here, right?

Reply:
Yes. My mistake. I forgot to remove this.